greeting-service/src/main/resources/project-stages.yml

```diff
@@ -0,0 +1,15 @@
+nameService:
+  url: "http://wfswarm-cache-name:8080/api"
+infinispan:
+  host: cache-server
+  port: 11222
+---
```

Comment:
Please separate into individual `project-*.yml` files. Using a single `project-stages.yml` is deprecated.

Reply:
done